source/frontend/src/features/properties/map/acquisition/update/summary/models.ts

```diff
@@ -47,19 +48,20 @@ export class UpdateAcquisitionSummaryFormModel
       legacyFileNumber: this.legacyFileNumber,
       fileName: this.fileName,
       rowVersion: this.rowVersion,
-      assignedDate: this.assignedDate,
-      deliveryDate: this.deliveryDate,
+      assignedDate: stringToNull(this.assignedDate),
+      deliveryDate: stringToNull(this.deliveryDate),
+      completionDate: stringToNull(this.completionDate),
       fileStatusTypeCode: toTypeCode(this.fileStatusTypeCode),
       acquisitionPhysFileStatusTypeCode: toTypeCode(this.acquisitionPhysFileStatusType),
       acquisitionTypeCode: toTypeCode(this.acquisitionType),
       regionCode: toTypeCode(Number(this.region)),
       project:
         this.project?.id !== undefined && this.project?.id !== 0
-          ? { ...defaultProject, id: this.project?.id }
+          ? { ...defaultProject, id: this.project?.id, rowVersion: 0 } // TODO: Fix this workaround when bug PSP-5871 gets fixed
           : undefined,
       product:
         this.product !== undefined && this.product !== 0
-          ? { ...defaultProduct, id: Number(this.product) }
+          ? { ...defaultProduct, id: Number(this.product), rowVersion: 0 } // TODO: Fix this workaround when bug PSP-5871 gets fixed
```

Comment:
Bringing this to other devs attention. This is a workaround until bug PSP-5871 gets fixed